front/src/Connexion/EmoteEventStream.ts

```diff
@@ -0,0 +1,19 @@
+import {Subject} from "rxjs";
+
+interface EmoteEvent {
+    userId: number,
+    emoteName: string,
+}
+
+class EmoteEventStream {
+
+    private _stream:Subject<EmoteEvent> = new Subject();
+    public stream = this._stream.asObservable();
+
+
+    onMessage(userId: number, emoteName:string) {
```

Comment:
The `onXXX` naming convention is typically used to register callbacks (to listen to something).

What you are doing here is "firing" an event.

So maybe `fireEvent` would be a better name?